packages/patternfly-4/content/design-guidelines/usage-and-behavior/wizard/design.md

```diff
@@ -2,33 +2,45 @@
 path: "/design-guidelines/usage-and-behavior/wizard"
 ---
 # Wizard
-Wizards provide a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome.
+Wizards provide a guided workflow that offers a path to complete a task, create an object or objects, or finish a series of steps for some other outcome. Wizards should incite trust in the user and guide them through an otherwise overwhelming experience.
 
-## Usage
 Use a wizard when
 
-* A system or product requires a set of information to be input by a user in a standard fashion. The guided stepped approach helps to eliminate errors of omission and catch them before having to process the entire chunk of information.
+* The task is long/complex and benefits from being broken into smaller more manageable steps. This allows the user to focus on smaller tasks and not feel overwhelmed.
+* There is a known step by step order of tasks, and these steps can be broken up into clearly defined categories/sections which can be indicated via labels on the steps.
+* This process is prescriptive. The actions a user takes in one step influence downstream steps.
 * A simple form is not sufficient to aid a user in completing a task
-* Your users are inexperienced and you do not want to overwhelm them with complexity.
 
 Avoid using a wizard for simple data entry tasks in which a basic form would suffice.
 
-The PatternFly wizard is designed as a modal experience. This will keep users focused on the task at hand. They must either complete all of the steps of the wizard or cancel before navigating elsewhere within an application.
+## Types of wizards
+Wizards can be modal or placed within the content area of a page.
+
+### Modal wizards
+![modal wizard](./img/wizard-progressive-step3-new.png)
+
+The modal wizard lives in a modal dialog. The width and height of the modal is variable and should be sized to comfortably fit the contents of the wizard at typical desktop screen resolutions. In most cases, the modal wizard is recommended as it will keep users focused on the task at hand. Users must either complete all of the steps of the wizard or cancel before navigating elsewhere within an application.
```

Comment:
Should we say that the wizard's height/width is variable? We have a standard size modal wizard and a full page modal wizard. They sizes could be adjusted by overriding the variables but do want to recommend that?

Reply:
Good question @matthewcarleton One of the reasons that I thought it was important to point this out was that often folks think the modal wizard won't work for them because the size is too constrained. But it does not have to be. They can always override these variables if they need to. @kybaker @mcoker do you have any thoughts about this?

Reply:
Ya that makes sense. Maybe we should just clarify that it has a full page modifier as well as the default size to clarify.

Reply:
OK. I made a small change that hopefully makes this more apparent.